spacy/tests/regression/util_add_marker.py

```diff
@@ -0,0 +1,32 @@
+import re
```

Comment:
I am not sure if this should even be committed... 🤔

Reply:
No, this is only going to be run once, so I don't think it should be committed.

Reply:
Removed in https://github.com/explosion/spaCy/pull/9629/commits/909177589dcdbde1cd4770f9f744d4d57d08d7e0